src/saga/adaptors/condor/condorjob.py

```diff
@@ -899,7 +904,7 @@ def _job_get_info_bulk(self, cluster_id, cluster_size, rm):
         if ret != 0:
             raise Exception("condor_q failed")
 
-        results = filter(None, out.split('\n'))
+        results = [_f for _f in out.split('\n') if _f]
```

Comment:
This would also remove empty lines.  Not sure if this is a problem or not, but needs checking.